deploy/operator.yaml

```diff
@@ -19,13 +19,13 @@ spec:
             - /usr/local/bin/ao-logs
             - /tmp/ansible-operator/runner
             - stdout
-          image: quay.io/alaypatel07/etcd-ha-operator:v0.6.0
+          image: quay.io/alaypatel07/etcd-ha-operator:v0.9.0
           volumeMounts:
             - mountPath: /tmp/ansible-operator/runner
               name: runner
               readOnly: true
         - name: etcd-ha-operator
-          image: quay.io/alaypatel07/etcd-ha-operator:v0.6.0
+          image: quay.io/alaypatel07/etcd-ha-operator:v0.9.0
```

Comment:
We should get the process of getting this image properly built started. At a min, we should have reminders that this needs to be changed.

Reply:
+1. I'll add a TODO and start digging into the process of properly building the image

Reply:
do we need to bump dirs as well? https://github.com/openshift/etcd-ha-operator/tree/master/etcd-ha-manifests/0.6.0

Reply:
Yes, we need to. But I was thinking of doing that in a separate PR. The manifests need to be tested in the OLM environment and I have not been able to spend time on them yet. 

Created #2 to track it.